plugins/modules/database/saphana/hana_query.py

```diff
@@ -0,0 +1,208 @@
+        type: str
+    database:
+        description: Define the database on which to connect.
+        type: str
+    encrypted:
+        description: Use encrypted connection. Defaults to false.
+        type: bool
+        default: false
+    filepath:
+        description:
+        - SQL query in file to run. Multiple files can be passed using YAML list syntax.
+        - Must be a string or list containing strings.
+        type: list
+    query:
+        description:
+        - SQL query to run. Multiple queries can be passed using YAML list syntax.
+        - Must be a string or list containing strings.
+        type: list
+
+author:
+    - Rainer Leber (@rainerleber)
+'''
+
+EXAMPLES = r'''
+
+- name: Simple select query
+  community.general.hana_query:
+    sid: "hdb"
+    instance: "01"
+    password: "Test123"
+    query: "select user_name from users"
+
+- name: Run several queries
+  community.general.hana_query:
+    sid: "hdb"
+    instance: "01"
+    password: "Test123"
+    query:
+    - "select user_name from users"
+    - select user_name from users
+    host: "localhost"
+    autocommit: False
+
+- name: Run several queries from file
+  community.general.hana_query:
+    sid: "hdb"
+    instance: "01"
+    password: "Test123"
+    filepath:
+    - /tmp/HANA_CPU_UtilizationPerCore_2.00.020+.txt
+    - /tmp/HANA.txt
+    host: "localhost"
+
+
+'''
+
+import io
+import csv
+import json
+from ansible.module_utils.basic import AnsibleModule
+
+
+def csv_to_json(rawcsv):
+    lines = rawcsv[:rawcsv.rfind('\n')]
+    reader_raw = csv.DictReader(io.StringIO(lines))
+    reader = [dict((k, v.strip()) for k, v in row.items()) for row in reader_raw]
+    tolist = list(reader)
+    return json.dumps(tolist)
+
+
+def main():
+    global module
+
+    module = AnsibleModule(
+        argument_spec=dict(
+            sid=dict(type='str', required=True),
+            instance=dict(type='str', required=False),
+            encrypted=dict(type='bool', required=False, default=False),
+            host=dict(type='str', required=False),
+            user=dict(type='str', required=False, default="SYSTEM"),
+            password=dict(type='str', required=True, no_log=True),
+            database=dict(type='str', required=False),
+            query=dict(type='list', required=False),
+            filepath=dict(type='list', required=False),
+            autocommit=dict(type='bool', required=False, default=True)
+        )
+    )
+    rc, out, err, out_raw = [0, "", "", ""]
+
+    params = module.params
+
+    sid = (params['sid']).upper()
+    instance = params['instance']
+    user = params['user']
+    password = params['password']
+    autocommit = params['autocommit']
+    host = params['host']
+    database = params['database']
+    encrypted = params['encrypted']
+
+    filepath = params['filepath']
+    query = params['query']
+
+    bin_path = f"/usr/sap/{sid}/HDB{instance}/exe/hdbsql"
```

Comment:
This does not work with Python 2.
```suggestion
    bin_path = "/usr/sap/{sid}/HDB{instance}/exe/hdbsql".format(sid=sid, instance=instance)
```